src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -56,10 +56,15 @@ def format_comment(issues: List[PullRequestIssue]):
     def format_subtitle(subtitle):
         return subtitle[:47] + "..." if len(subtitle) > 50 else subtitle
 
+    def format_url(url):
+        return url + "?referrer=github-pr-bot"
```

Comment:
@leedongwei  Do we have a central location where we store referrer keys?

Reply:
If we don't let's create one with at least this one and keep adding to it. It helps making analysis much easier.

Reply:
where are we creating this?

Reply:
just put it under `src/sentry/types/referrer.py`